system/handlers/admin/DataHelpers.cfc

```diff
@@ -153,15 +158,19 @@ component extends="preside.system.base.adminHandler" {
 		var baseLink      = event.buildadminLink( objectName=relatedObject, recordId="{recordId}" );
 		var list          = [];
 
-		for( var record in records ) {
-			if ( baseLink.len() ) {
-				list.append( '<a href="#( baseLink.replace( '{recordId}', record.id ))#">#record.label#</a>' );
+		for ( var record in records ) {
+			var recordLabel = record.label;
+
+			recordLabel = _renderyNoValue( objectName=objectName, propertyName=propertyName, propertyValue=recordLabel );
```

Comment:
Am not sure why `_renderyNoValue()` is required here? - we know that we have a list of records, so there should always be something to render?

Reply:
Removed.